libs/ui/src/admin/StackExample/StackExample.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import { Onboarding } from './StackExample'
```

Comment:
I would give the component name the same as the filename to avoid confusion.

```suggestion
import { StackExample } from './StackExample'
```